### PR TITLE
Add GRO space

### PR DESCRIPTION
**Pull request by Mohsen-T**
**Add space for GrowthDefi**  
  
Etherscan: https://etherscan.io/token/0x09e64c2B61a5f1690Ee6fbeD9baf5D6990F8dFd0 
CoinGecko: https://www.coingecko.com/en/coins/growth-defi
Website: https://growthdefi.com 
Twitter: https://twitter.com/GrowthDefi

### CODE REVIEW

File: spaces/GrowthDefi/index.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    }
+  ],
+  "members": [ // List of official addresses that can post in "Core" tab of the space
+    "0xeF8305E140ac520225DAf050e2f71d5fBcC543e7" // Core member address
+  ],
+  "filters": { // Filters
+    "defaultTab": "all", // The default tab for the space
+    "minScore": 1, // Minimum balance from the base token that a user should have to show his proposal in the space (unless the address is a core address)
+    "onlyMembers": true // Shows only core tab and core proposals
+    "invalids": [ // List of proposals IDs (use this to remove a proposal from your space)
```
Comment (gokaiorg) [inline spaces/GrowthDefi/index.json:24]: JSON standard does not allow comments. Please remove them.

File: skins/GrowthDefi.scss [hunk 2]
```diff
@@ -0,0 +1,10 @@
+.GrowthDefi {
```
Comment (gokaiorg) [inline skins/GrowthDefi.scss:1]: The css selector should similar to the space folder and do not contain uppercases.
```suggestion
.growthdefi {
```

**Review by gokaiorg — CHANGES_REQUESTED** [review]
Do no add content in another space:
Delete `spaces/aave/logo - Copy.png`.

No uppercase allowed in file name:
Replace `skins/GrowthDefi.scss` to `skins/growthdefi.scss`

No uppercase allowed in space folder:
Replace `spaces/GrowthDefi/index.json` to `spaces/growthdefi/index.json`

File: skins/GrowthDefi.scss (hunk 2 above)
Comment (Mohsen-T) [reply skins/GrowthDefi.scss]: Changed to growthdefi.

File: spaces/GrowthDefi/index.json (hunk 1 above)
Comment (Mohsen-T) [reply spaces/GrowthDefi/index.json]: Removed comments.

Comment (Mohsen-T) [reply spaces/GrowthDefi/index.json]: Also removed unnecessary file in the aave folder. And changed the uppercases to lowercases. Thanks.

File: spaces/growthdefi/index.json [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+  "members": [ 
+    "0xeF8305E140ac520225DAf050e2f71d5fBcC543e7"
+  ],
+  "filters": { 
+    "defaultTab": "all", 
+    "minScore": 1, 
+    "onlyMembers": true 
+    "invalids": [ 
+      
+    ]
```
Comment (gokaiorg) [inline spaces/growthdefi/index.json:26]: You should delete `invalids` field.
```suggestion
```

**Review by gokaiorg — CHANGES_REQUESTED** [review]

File: spaces/growthdefi/index.json (hunk 3 above)
Comment (Mohsen-T) [reply spaces/growthdefi/index.json]: Updated.

File: spaces/growthdefi/index.json [hunk 4]
```diff
@@ -0,0 +1,26 @@
+{
+  "name": "growthdefi",
```
Comment (gokaiorg) [inline spaces/growthdefi/index.json:2]: You could use uppercase in the name if needed.

**Review by gokaiorg — CHANGES_REQUESTED** [review]
The colors used in your skin make the interface unreadable, please change with others or remove the skin to use the default theme.
image
image

**Review by gokaiorg — APPROVED** [review]

File: spaces/growthdefi/index.json (hunk 4 above)
Comment (Mohsen-T) [reply spaces/growthdefi/index.json:2]: Please check the updates on the code if that does a better job.

**Pull request merged**